### PR TITLE
:sparkles: Add toggle to disable fast scroller

### PR DESCRIPTION
**Pull request by Maxr1998**
As discussed in #522.

### CODE REVIEW

File: lib/components/LayoutSettingsScreen/FastScrollSelector.dart [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<Box<FinampSettings>>(
+      valueListenable: FinampSettingsHelper.finampSettingsListener,
+      builder: (_, box, __) {
+        return SwitchListTile.adaptive(
+          title: Text(AppLocalizations.of(context)!.showFastScroller),
+          value: FinampSettingsHelper.finampSettings.showFastScroller,
+          onChanged: (value) => FinampSettingsHelper.setShowFastScroller(value),
```
Comment (Chaphasilor) [inline lib/components/LayoutSettingsScreen/FastScrollSelector.dart:19]: A `subtitle` with a short description of what the "fast scroller" is would be nice here :)

**Comment by Chaphasilor** [comment]
You forgot to use the [arcane arts](https://github.com/jmshrv/finamp/blob/main/CONTRIBUTING.md#the-arcane-arts-code-generation), so the setting wasn't actually being saved across restarts :)  
I've quickly done that for you, everything else looks good!

Thanks a ton!

**Comment by Chaphasilor** [comment]
Forgot about the subtitle, never mind then...  
I should go to bed 🙃

**Comment by Maxr1998** [comment]
Ah yes, the arcane arts.. 🫠
Thanks for handling those for me! About the summary, I didn't think one was necessary, but if it creates any confusion we can still add one.

**Pull request merged**